ote_cli/ote_cli/utils/validate_path.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import os
+import errno
+
+
+def is_path_valid(test_string: str) -> bool:
+    try:
+        if not isinstance(test_string, str) or not test_string:
+            return False
+        _, test_string = os.path.splitdrive(test_string)
+
+        root = os.path.sep
+        assert os.path.isdir(root)
+
+        root = root.rstrip(os.path.sep) + os.path.sep
+        for pathname_part in test_string.split(os.path.sep):
+            try:
+                os.lstat(root + pathname_part)
```

Comment:
I think that if I use `test_string=/home/lbeynens/somefolder/somefile.txt`, then this cycle will check paths `/home`, `/lbeynens`, `/somefolder` -- and obviously will fail for all paths except the first one.
Is this correct?